zest/releaser/utils.py

```diff
@@ -494,6 +518,47 @@ def system(command, input=''):
     return result
 
 
+def retry_command(command):
```

Comment:
Yeah, the name is not optimal.

The way I would do an `allow_retry` option, would be to split the `system` command into one function that handles the retry bit, and a second function that focuses on just running the command once, otherwise the amount of logic in the `system` function would get a bit too much for my comfort. But then you have the same situation that there is now in this pull request, except maybe different names.

`system` is not very informative, so `execute_command` may indeed be nicer.

So I give two proposals:

1. Keep current code, rename `system` to `execute_command`, rename `retry_command` to `execute_command_with_retry`.

2. Add `allow_retry` option to `system`. I would say default False, because we only need it in a few cases. To prevent one function from getting too big and difficult, rename current `system` to `_execute_command_once`. Rename `retry_command` to `system` (or `execute_command` for clarity) and give that function the extra option.

So for number 2:

```
def _execute_command(command):
    do what current system(command) does

def execute_command(command, allow_retry=False):
    if not allow_retry:
        return _execute_command(command)
    do what current retry_command(command) does
```